api/spec/rails_helper.rb

```diff
@@ -26,15 +32,40 @@
 # If you are not using ActiveRecord, you can remove these lines.
 begin
   ActiveRecord::Migration.maintain_test_schema!
-rescue ActiveRecord::PendingMigrationError => e
+rescue ActiveRecord::PendingMigrationErrorRAI => e
   abort e.to_s.strip
 end
 RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_paths = [
     Rails.root.join('spec/fixtures')
   ]
 
+  Dir[Rails.root.join("spec/support/**/*.rb")].sort.each { |f| require f }
+
+  config.include AuthHelpers
```

Comment:
AuthHelpers is included without checking if it is defined. Add a check or rescue to avoid errors if the module is missing.
```
config.include AuthHelpers if defined?(AuthHelpers)
```